media-sound/helm/helm-9999.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="A free polyphonic synth with lots of modulation"
+HOMEPAGE="http://tytel.org/helm/"
+LICENSE="GPL-3+ CC-BY-3.0"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mtytel/helm.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/mtytel/helm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+SLOT="0"
+
+IUSE="standalone lv2 vst"
+REQUIRED_USE="|| ( standalone lv2 vst )"
```

Comment:
Mainly because vst used to be problematic because of the SDK dependency.
Also, it's an opinionated choice since we have LV2 on Linux I don't see much reason to use/build VST.

But you're right, it probably makes more sense to test as much as possible and since the VST SDK is included in the sources I don't expect it to cause a lot of issue, so I'll add it to the test USE config as well.